throughs/flatten.js

```diff
@@ -33,9 +33,7 @@ FlattenStream.prototype.resume = function (data) {
     while(!this.sink.paused && this.queue.length)
       this.sink.write(this.queue.shift())
     if(!this.queue.length) {
-      else {
```

Comment:
This `else` was invalid JS syntax.

Reply:
woah, I think I remember you showed me this once ;)

Reply:
it must never have reached this line otherwise it would have blown up with syntaxerror